stolon/templates/keeper-ro-service.yaml

```diff
@@ -16,5 +15,7 @@ spec:
     - port: {{ .Values.ports.externalPort }}
       targetPort: {{ .Values.ports.internalPort }}
   selector:
-    app: {{ template "stolon.keeper.fullname" . }}
+    app: "stolon-keeper"
+    release: "{{ .Release.Name }}"
     component: keeper
```

Comment:
you did not delete component from label selector, but did delete it from the labels in keeper.

Reply:
No I did not. Component is still in place in all pod specs which are used for selection. I only deleted the cluster-name as well as component from the StatefulSets and according resources. K8 service discovery works fine for keepers as is.